### PR TITLE
Allow empty lesson groups

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Completes [PLAT-1065](https://codedotorg.atlassian.net/browse/PLAT-1065). Removes the protections against empty lesson groups.

The script overview page shows this for a lesson group with no lessons:
Screen Shot 2021-06-23 at 8 48 40 AM

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by bethanyaconnor** [comment]
The only failing test succeeds locally, so I'm waiting for another Drone run before investigating further.

File: dashboard/test/dsl/script_dsl_test.rb [hunk 1]
```diff
@@ -722,6 +722,70 @@ def setup
     assert_equal expected, output
   end
 
+  test 'Script DSL for lesson group without lesson' do
```
Comment (bethanyaconnor) [inline dashboard/test/dsl/script_dsl_test.rb:725]: I didn't change any behavior here but figured a unit test was worth adding as we allow this scenario now

**Review by dmcavoy — APPROVED** [review]
Thanks for following up on this!

**Review by davidsbailey — APPROVED** [review]
nice improvement!

**Pull request merged**